client/src/panels/panel_declarations/services/single_service_panels/service_standards.js

```diff
@@ -0,0 +1,167 @@
+import { Fragment } from "react";
+import text from "../services.yaml";
+import FootNote from "../../../../models/footnotes/footnotes.js";
+import {
+  create_text_maker_component,
+  Panel,
+  DisplayTable,
+  FilterTable,
+} from "../../../../components";
+import { newIBCategoryColors, businessConstants } from "../../shared.js";
+import { IconAttention, IconCheck } from "../../../../icons/icons.js";
+
+const { text_maker, TM } = create_text_maker_component(text);
+
+const standard_statuses = ["met", "not_met"];
+const color_scale = d3
+  .scaleOrdinal()
+  .domain(standard_statuses)
+  .range(_.take(newIBCategoryColors, 2));
+
+export class ServiceStandards extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      active_statuses: standard_statuses,
+    };
+  }
+
+  render() {
+    const { service } = this.props;
+    const { active_statuses } = this.state;
+    const { result_simple_statuses } = businessConstants;
+    const standards = service.standards;
+
+    const footnotes = _.chain(standards)
```

Comment:
Not sure if this would be enough for Footnotes (same for service_digital_status)